tests/test_user_mgt.py

```diff
@@ -26,6 +26,11 @@
 from firebase_admin import _user_mgt
 from tests import testutils
 
+try:
```

Comment:
Import via `six`:

```
from six.moves.urllib import parse

parse.urlsplit()
parse.parse_qsl()
```

Reply:
This one gives pylint issue

```
$ ./lint.sh all
Running linter on module firebase_admin
Running linter on module tests
************* Module tests.test_user_mgt
E: 29, 0: Unable to import 'six.moves.urllib' (import-error)
```

Do you think upgrading pylint will solve issue?

Reply:
fixed